craigslist-apartments.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+let axios = require('axios');
+let cheerio = require('cheerio');
+let filters = require('./filters');
+
+const Filter = new filters();
```

Comment:
We want the handler to specify the filters, not this file.

Reply:
Done.